flashinfer/api_logging.py

```
@@ -594,6 +764,81 @@ def _dump_function_outputs(dump_dir: str, result: Any) -> None:
+                continue
+            if entry["format"] == "safetensors":
+                from safetensors.torch import save_file  # noqa: PLC0415
+
+                save_file(
+                    {k: v.contiguous() for k, v in tensors.items()},
+                    str(dump_path / f"{kind}.safetensors"),
+                )
+            else:
+                torch.save(tensors, dump_path / f"{kind}.pt")
+            flushed += 1
+        except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)